src/libcollections/btree/node.rs

```diff
@@ -970,16 +891,12 @@ impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Internal>, marker::
             // This cast is a lie, but it allows us to reuse the key/value insertion logic.
             self.cast_unchecked::<marker::Leaf>().insert_fit(key, val);
 
-            slice_insert(
-                slice::from_raw_parts_mut(
-                    self.node.as_internal_mut().edges.as_mut_ptr(),
-                    self.node.len()
-                ),
-                self.idx + 1,
-                edge.node
-            );
+            slice_insert(slice::from_raw_parts_mut(self.node.as_internal_mut().edges.as_mut_ptr(),
+                                                   self.node.len()),
+                         self.idx + 1,
+                         edge.node);
 
-            for i in (self.idx+1)..(self.node.len()+1) {
+            for i in (self.idx + 1)..(self.node.len() + 1) {
```

Comment:
Something that I don't think that `rustfmt` wanted to touch here but is a style question which this file is inconsistent about is whether there should be parens here. I don't believe they are strictly necessary (there is somewhere else in this file without them), but I personally find that they make things clearer.